lighthouse-core/gather/gather-runner.js

```diff
@@ -75,17 +75,17 @@ class GatherRunner {
    * will always represent the post-redirected URL. options.initialUrl is the
    * pre-redirect starting URL.
    * @param {!Driver} driver
-   * @param {!Object} options
+   * @param {!Object} passContext
    * @return {!Promise}
    */
-  static loadPage(driver, options) {
-    return driver.gotoURL(options.url, {
+  static loadPage(driver, passContext) {
```

Comment:
I guess if I were being real about it, I'd leave `gotoURL` as `options` that is `{waitForLoad: boolean, passContext: Object}` rather than abuse passContext, but the layer of nesting was annoying. I'm guessing you're in favor of doing that instead? :)

Reply:
ha, it is more awkward but probably better. It'll get better when we pull out a `passContext` type :)

Reply:
done 👍